libmamba/data/micromamba.nu

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nu
+def-env "micromamba activate"  [name: string] {
```

Comment:
This is missing some functionality:

- `micromamba activate /path/to/env`
- Adding condabin, see https://github.com/mamba-org/mamba/blob/2b0c975429679bc728fead20d2b4cf264b25c5e7/libmamba/data/mamba.fish#L1-L4

Reply:
@jonashaag  are you referring to the condabin? Or does micromamba also take an env as an input?

Reply:
This is fixed @jonashaag should be ready now the PR!